com.unity.render-pipelines.universal/Shaders/SimpleLit.shader

```diff
@@ -245,6 +245,7 @@ Shader "Universal Render Pipeline/Simple Lit"
 
             // -------------------------------------
             // Material Keywords
+            #pragma multi_compile SIMPLE_LIT
```

Comment:
Is this supposed to be be #define SIMPLE_LIT, otherwise I don't get why we are adding a multi_compile for simple lit.

Reply:
This was leftover when splitting the DepthNormals passes. Should not be there.
Removing in next commit.